docs/tutorials/add-route.md

```diff
@@ -0,0 +1,218 @@
+# Adding route 🛡
+
+This document is part of the Appwrite contributors' guide. Before you continue reading this document make sure you have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/master/CODE_OF_CONDUCT.md) and the [Contributing Guide](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md).
+
+### 1. Alias
+Setting an alias  is used to permit access the route from an alias url as well,
+second parameter is used to push default values to the route.
```

Comment:
```suggestion
The first parameter specifies the alias URL, the second parameter specifies default values for route parameters. 
```

Reply:
I think the `:bucketId` might actually be called a slug:

https://en.wikipedia.org/wiki/Clean_URL#Slug

Reply:
Or this name =_- I don't actually know which is best. Pick your favorite

https://swagger.io/docs/specification/describing-parameters/#path-parameters